index.html

```diff
@@ -0,0 +1,204 @@
+        <p class="venue">
+          @ National Museum of Korea,Art Center Nabi and more
+        </p>
+      </div>
+    </div>
+  </section>
+
+  <section class="second-section">
+    <div class="second-section-container">
+      <h2 class="second-section-heading">Main <span>Pro</span>gram</h2>
+
+      <div class="program-list">
+        <div class="first-block flex-box">
+          <img class="icons" src="images/program1.png" alt="" />
+          <h2>Lecture</h2>
+          <p>
+            Listen to the spekaer from various countries about the messages of
+            sharing and opening.
+          </p>
+        </div>
+
+        <div class="second-block flex-box">
+          <img class="icons" src="images/program2.png" alt="" />
+          <h2>CC Exhibiton</h2>
+          <p>
+            Appreciate various creations applying CC licence of
+            artists,organized from Art Center Nabi.
+          </p>
+        </div>
+
+        <div class="third-block flex-box">
+          <img class="icons" src="images/program3.png" alt="" />
+          <h2>Forum</h2>
+          <p>
+            Have the time to share your thoughts and opinions with experts for
+            each topic.
+          </p>
+        </div>
+
+        <div class="fourth-block flex-box">
+          <img class="icons" src="images/program4.png" alt="" />
+          <h2>Workshop</h2>
+          <p>
+            Try creating your own work using open source licence rather than
+            just watching.
+          </p>
+        </div>
+
+        <div class="fifth-block flex-box">
+          <img class="icons" src="images/program5.png" alt="" />
+          <h2>CC Ignite</h2>
+          <p>
+            Get opportunities to network with CC affiliates around the
+            world,also after the summit.
+          </p>
+        </div>
+      </div>
+      <div class="button-container">
+        <button type="button" class="btn btn-primary btn-lg btn-block">
+          Join SD Global Summit 2021
+        </button>
+      </div>
+      <a class="btn-projects" href="#">SEE WHOLE PROGRAM</a>
+    </div>
+  </section>
+
+  <div class="section-speakers-heading">
+    <h2>Feature<span>d S</span>peakers</h2>
+
+  </div>
+
+  <section class="partners-section" id="partners">
+    <h2 class="partners-heading"><strong>Pa<span>rtn</span>ers</strong></h2>
+
+    <div class="partners-flex">
+      <div class="partners-list">
+        <img class="partner-image" src="images/partner1.png" alt="" />
+      </div>
+      <div class="partners-list">
+        <img class="partner-image" src="images/partner2.png" alt="" />
+      </div>
+      <div class="partners-list">
+        <img class="partner-image" src="images/partner3.png" alt="" />
+      </div>
+      <div class="partners-list">
+        <img class="partner-image" src="images/partner4.png" alt="" />
+      </div>
+      <div class="partners-list">
+        <img class="partner-image" src="images/partner5.png" alt="" />
+      </div>
```

Comment:
```suggestion
        <img class="partner-image" src="images/partner1.png" alt="" />
      </div>
      <div class="partners-list">
        <img  src="images/partner2.png" alt="" />
      </div>
      <div class="partners-list">
        <img  src="images/partner3.png" alt="" />
      </div>
      <div class="partners-list">
        <img src="images/partner4.png" alt="" />
      </div>
      <div class="partners-list">
        <img src="images/partner5.png" alt="" />
      </div>
```
In your CSS, this image can be accessed for styling by calling

.partners-list img {
/* desired style rules*/
}

This again helps with the DRY CSS experience to help your capstone project shine out when recruiters view your codease and see how neat and easy to read it is. it also helps collaborators and contributors.